src/models/multiChart.js

```diff
@@ -398,13 +395,60 @@ nv.models.multiChart = function() {
               }
             }
 
-            function highlightPoint(serieIndex, pointIndex, b){
-              for(var i=0, il=charts.length; i < il; i++){
-                var chart = charts[i];
-                try {
-                  chart.highlightPoint(serieIndex, pointIndex, b);
-                } catch(e){}
-              }
+            function highlightPoint(series, pointIndex, b, pointYValue) {
```

Comment:
This change corrects a bug with highlighting points where the wrong series got highlighted.

Reply:
I think this may have been fixed by a different PR I just merged.

Reply:
@liquidpele are you sure? The only recent change I can see related to highlighting points is https://github.com/novus/nvd3/commit/89567800f575f4df75c0bdc3a3127d1bbc1ef2b6 in scatter chart. That fix does not fix the bug here. The bug is specifically related to multi chart.

Reply:
The problem was that the existing code would highlight multiple charts.

e.g. Chart has line plus bar. User has highlighted first point in line chart: pointIndex is 0, seriesIndex is 1.

The current code would highlight point 0 in lines AND in bars. 

It shouldn't do that. It should only highlight a point in the chart it belongs to.

Reply:
cool